libvcx/src/api_lib/api_c/credential.rs

```diff
@@ -1070,7 +1069,7 @@ mod tests {
         let _setup = SetupMocks::init();
 
         let handle = credential::credential_create_with_offer("test_send_request", ARIES_CREDENTIAL_OFFER).unwrap();
-        assert_eq!(credential::get_state(handle).unwrap(), VcxStateType::VcxStateRequestReceived as u32);
+        assert_eq!(credential::get_state(handle).unwrap(), HolderState::OfferReceived as u32);
```

Comment:
Let's improve this test and add state assert after wee call `vcx_credential_send_request`

Reply:
Actually maybe we can just delete this test. This test is subset of of  `test_vcx_credential_update_state`